certbot/tests/error_handler_test.py

```diff
@@ -112,6 +113,59 @@ def test_sysexit_ignored(self):
             pass
         self.assertFalse(self.init_func.called)
 
+    @mock.patch('certbot.main.sys')
+    def test_handle_exception(self, mock_sys):
```

Comment:
It's confusing to me to have `_handle_exception()` defined in `main.py` but tested in `error_handler_test.py`. I don't know the history of why it's in `main`, but could we move it to `error_handler.py` to match the test?